leo_docking/CMakeLists.txt

```diff
@@ -21,7 +21,6 @@ ament_python_install_package(${PROJECT_NAME})
 
 # Install Python executables
 install(PROGRAMS
-  leo_docking/scripts/docking_client.py
```

Comment:
Could try to default to C++17 in CMakeLists as that is the default for humble. Cannot comment on the line where I would like to point about this... Add into this file (usually close to top) and into the CMakeLists of the msgs package

```# Default to C++17
if(NOT CMAKE_CXX_STANDARD)
  set(CMAKE_CXX_STANDARD 17)
endif()
```